src/parse_instruction.py

```diff
@@ -274,6 +273,16 @@ def parse_word(elems: List[str], valid: str = valid_word) -> str:
     return S
 
 
+def parse_quoted(elems: List[str], quote_chars: str) -> str:
+    S: str = ""
+    while elems and elems[0] not in quote_chars:
+        # Handle backslash-escaped characters
+        if elems[0] == "\\":
+            elems.pop(0)
+        S += elems.pop(0)
```

Comment:
return early if not elems, to leave (currently somewhat non-existent) unterminated string literal error handling to the caller

Reply:
I think I fixed what you were asking about? But if not, I can push another commit